src/renderer/containers/query-browser.jsx

```diff
@@ -269,7 +270,7 @@ class QueryBrowserContainer extends Component {
             <ResizableBox className="react-resizable react-resizable-ew-resize"
               width={235}
               minConstraints={[235, 300]}
-              maxConstraints={[750, 10000]}>
+              maxConstraints={[750, 10000]} style={STYLES.resizeable}>
```

Comment:
`ResizableBox` does not support `style` property. Actually it is crashing the `ResizableBox`.